src/Akeneo/Platform/Bundle/ImportExportBundle/Resources/translations/messages.fi_FI.yml

```
@@ -77,7 +77,7 @@ pim_importexport:
       index: Näytä tuontiprofiilien lista
       create: Tuontiprofiilin luonti
       show: Näytä tuontiprofiili
-      edit: Editoi tuontiprofiilia
+      edit: Muokkaa tuontiprofiilia
```

```suggestion
      edit: Muokkaa tuontiprofiilia
```

I haven't done any pull up since ages but are you sure it should be included ?

---

Seems fine according to @pierallard https://akeneo.slack.com/archives/C031QDW61/p1661786784601599?thread_ts=1661785742.525729&cid=C031QDW61